main.py

```diff
@@ -90,7 +90,7 @@
   optimiser.share_memory()
   env.close()
 
-  fields = ['T_Value', 'Reward', 'Avg_Steps', 'time']
+  fields = ['t', 'rewards', 'avg_Steps', 'time']
```

Comment:
`avg_steps`

Reply:
All changes have been included in the code.